code/modules/clothing/spacesuits/hardsuit.dm

```diff
@@ -787,7 +787,25 @@
 	allowed = list(/obj/item/gun, /obj/item/ammo_box, /obj/item/ammo_casing, /obj/item/melee/baton, /obj/item/melee/transforming/energy/sword/saber, /obj/item/restraints/handcuffs, /obj/item/tank/internals)
 	helmettype = /obj/item/clothing/head/helmet/space/hardsuit/shielded/syndi
 	slowdown = 0
+	shield_state = "shield-red"
+	shield_on = "shield-red"
+
+/obj/item/clothing/suit/space/hardsuit/shielded/syndi/attackby(obj/item/W, mob/living/user, params)
+	if(W.tool_behaviour == TOOL_MULTITOOL)
+		if(shield_state == "shield-red")
+			shield_state = "shield-old"
+			shield_on = "shield-old"
```

Comment:
This makes the shield overlay show up, even if there are no charges and it therefore shouldn't.

Reply:
it is fixed